docs/installation.md

```diff
@@ -0,0 +1,100 @@
+# nf-core/neutronstar: Installation
+
+## Installation
```

Comment:
Most of this file is on the website, and the docs readme file links there. I think we should take out all of the generic stuff here and link to the website, and update the docs readme file to still link to this page. Make sense?

Reply:
Makes perfect sense imo